.circleci/config.yml

```diff
@@ -0,0 +1,39 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/configuration-reference
+version: 2.1
+
+# Define a job to be invoked later in a workflow.
+# See: https://circleci.com/docs/configuration-reference/#jobs
+jobs:
+  build:
+    # Specify the execution environment. You can specify an image from Docker Hub or use one of our convenience images from CircleCI's Developer Hub.
+    # See: https://circleci.com/docs/configuration-reference/#executor-job
+    docker:
+      - image: cimg/openjdk:11.0.20
+    # Add steps to the job
+    # See: https://circleci.com/docs/configuration-reference/#steps
+    steps:
+      - checkout
+
+      - run:
+          name: "Maven Install"
+          command: "mvn -U clean install"
+
+      - store_artifacts:
+          path: ./core/target/surefire-reports
+          destination: surefire-reports
```

Comment:
Change `store_artifacts` to `store_test_results` and remove the `destination` field. That should publish all of the test results.

Reply:
Yes, I did that but got an error you can see it here: 
https://app.circleci.com/pipelines/github/nuodb/migration-tools/6/workflows/6d915c86-379b-457a-bd28-eb9feacb69c8/jobs/5

It didn't failed the build, just complained about the step.

Reply:
Okay, so that's what you meant when you said CircleCI does not support TestNG. Maybe there's a way to make TestNG spit out JUnit format.

Reply:
I'm not sure that the `testng-results.xml` is even necessary. Try changing the path to `./core/target/surefire-reports/junitreports` to avoid the problematic XML file.

Reply:
Yep, that worked: https://app.circleci.com/pipelines/github/nuodb/migration-tools/17/workflows/74e68094-f4c5-47ce-9e30-87b252a4f922/jobs/15/steps

Thanks for the help @adriansuarez